src/app/shared/components/ui/email-input/email-input.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="mt-4 ml-8 relative">
+<div class="flex items-center">
+<ng-content></ng-content>
```

Comment:
```suggestion
<ng-content></ng-content>
```
don't using here <ng-content/> using reusable input